INZFS.MVC/Models/DynamicForm/YesornoInputModel.cs

```diff
@@ -13,7 +13,7 @@ public class YesornoInputModel : BaseModel, IValidatableObject
     {
         protected override IEnumerable<ValidationResult> ExtendedValidation(ValidationContext validationContext)
         {
-            if (Mandatory == true)
+            if (Mandatory == true && MarkAsComplete)
```

Comment:
With && MarkAsComplete causes a null reference error when no option is selected. Remove or develop a new solution to handle null references.

Reply:
fixed